### PR TITLE
Update readme with contact details

### PR DESCRIPTION
**Pull request by alicenoakes**
So folks can say 👋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ See live examples of GOV.UK Frontend components, and guidance on when to use
 them in your service, in the [GOV.UK Design
 System](https://govuk-design-system-production.cloudapps.digital/).
 
+## Contact the team
+
+GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK
+Frontend, please [email the Design System team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them 
+[on Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
```
Comment (nickcolley) [inline README.md]: Could we say 'get in touch with us'?

Comment (igloosi) [reply README.md]: do we need to update the 

> 🚨 GOV.UK Frontend is in private beta and these instructions are subject to change. If you want to know more about how you can use GOV.UK Frontend, please get in touch with the Design System team at GDS. 🚨

at the top, considering latter part is similar. for example to only say  :

🚨 GOV.UK Frontend is in private beta and these instructions are subject to change. 🚨

**Comment by igloosi** [comment]
@nickcolley had to rebase with changes. is it still ok?

**Comment by nickcolley** [comment]
@igloosi yeah, lets get it in :)

**Pull request merged**